scipy/stats/tests/test_distributions.py

```diff
@@ -2969,6 +2969,56 @@ def test_entropy_transposed(self):
                                   stats.entropy(pk, qk, axis=1))
 
 
+class TestDifferentialEntropy(object):
+    """
+    Results are compared with the R package vsgoftest.
+        
+    """
+
+    def test_differential_entropy_base(self):
+        
+        random_state = np.random.RandomState(0)
+        values = random_state.standard_normal(100)
+        entropy = stats.differential_entropy(values)
+        assert_(abs(entropy - 1.122044) < 1.e-5)
+        
+        entropy = stats.differential_entropy(values, window_length=8)
+        assert_(abs(entropy - 1.349401) < 1.e-5)
+
+    def test_differential_entropy_base_2d_nondefault_axis(self):
+        random_state = np.random.RandomState(0)
+        values = random_state.standard_normal((3, 100))
+        
+        entropy = stats.differential_entropy(values, axis=1)
+        assert_array_almost_equal(
+            entropy,
+            [1.122044, 1.102944, 1.129616],
+        )
+        
+        entropy = stats.differential_entropy(values, axis=1, window_length=8)
+        assert_array_almost_equal(
+            entropy,
+            [1.349401, 1.338514, 1.292332],
+        )
+
+    def test_base_differential_entropy_with_axis_0_is_equal_to_default(self):
+        random_state = np.random.RandomState(0)
+        values = random_state.standard_normal((100, 3))
+        
+        entropy = stats.differential_entropy(values, axis=0)
+        default_entropy = stats.differential_entropy(values)
+        assert_array_almost_equal(entropy, default_entropy)
```

Comment:
the NumPy docs seem to recommend using i.e., `assert_allclose` for more consistent floating point comparisons: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_almost_equal.html

Reply:
Changed to `assert_allclose`.